### DISCUSSION TITLE
SWR config that should be set to X by default in your mind and why?

### DISCUSSION THREAD

**Discussion by huozhi** [vercel/swr#2032] [Ideas]
When you're using SWR hooks, mostly I think you'd love to see the hook and most of configs should work out of box by default. Sometimes you're trying to tune the config to make it work with your case. Share anything you think that changing to another value by default will make your SWR life easier, what's the case in your mind?

**Comment by shuding** [comment]
Maybe [`keepPreviousData`](https://github.com/vercel/swr/releases/tag/2.0.0-beta.1) should be `true` by default? :P
